doc/help/pypeit_chk_alignments.rst

```diff
@@ -6,8 +6,7 @@
     Display MasterAlignment image and the trace data
     
     positional arguments:
-      master_file      PypeIt Master Alignment file [e.g.
-                       MasterAlignment_A_1_01.fits]
+      master_file      PypeIt Master Alignment file [e.g. MasterAlignment_A_1_01.fits]
```

Comment:
It's annoying that these files get changed based on the width of the terminal at the instance when the docs are updated.  I'll look into fixing the width to 80 characters for these output files.

Reply:
if you can find the trick, i can try folding it in here and re-generating the docs so there aren't so many changes.

Reply:
I found the trick, but it's a bit annoying.  For each script, you have to change the formatter to something like the following:
```
    width = 78 if return_parser else None
    parser = argparse.ArgumentParser(description=run_pypeit_usage(),
                                     formatter_class=lambda prog:
                                         argparse.RawDescriptionHelpFormatter(prog, width=width))
```
That's for `run_pypeit`, but the details of the formatter class sometimes changes.  I suggest we leave this fix to a latter PR.

Reply:
ah, ok. thanks for looking into it.